shell/platform/darwin/ios/framework/Source/FlutterPlatformViewsTest.mm

```diff
@@ -1302,20 +1328,26 @@ - (void)testApplyBackdropFilterAPIChangedInvalidInputRadius {
 }
 
 - (void)testBackdropFilterVisualEffectSubviewBackgroundColor {
-  UIVisualEffectView* visualEffectView = [[UIVisualEffectView alloc]
-      initWithEffect:[UIBlurEffect effectWithStyle:UIBlurEffectStyleLight]];
-  PlatformViewFilter* platformViewFilter =
-      [[PlatformViewFilter alloc] initWithFrame:CGRectMake(0, 0, 10, 10)
-                                     blurRadius:5
-                               visualEffectView:visualEffectView];
-  CGColorRef visualEffectSubviewBackgroundColor = nil;
-  for (UIView* view in [platformViewFilter backdropFilterView].subviews) {
-    if ([NSStringFromClass([view class]) hasSuffix:@"VisualEffectSubview"]) {
-      visualEffectSubviewBackgroundColor = view.layer.backgroundColor;
+  __weak UIVisualEffectView* weakVisualEffectView;
+
+  @autoreleasepool {
+    UIVisualEffectView* visualEffectView = [[UIVisualEffectView alloc]
+        initWithEffect:[UIBlurEffect effectWithStyle:UIBlurEffectStyleLight]];
+    weakVisualEffectView = visualEffectView;
+    PlatformViewFilter* platformViewFilter =
+        [[PlatformViewFilter alloc] initWithFrame:CGRectMake(0, 0, 10, 10)
+                                       blurRadius:5
+                                 visualEffectView:visualEffectView];
+    CGColorRef visualEffectSubviewBackgroundColor = nil;
+    for (UIView* view in [platformViewFilter backdropFilterView].subviews) {
+      if ([NSStringFromClass([view class]) hasSuffix:@"VisualEffectSubview"]) {
+        visualEffectSubviewBackgroundColor = view.layer.backgroundColor;
+      }
     }
+    XCTAssertTrue(
+        CGColorEqualToColor(visualEffectSubviewBackgroundColor, UIColor.clearColor.CGColor));
   }
-  XCTAssertTrue(
-      CGColorEqualToColor(visualEffectSubviewBackgroundColor, UIColor.clearColor.CGColor));
+  XCTAssertNil(weakVisualEffectView);
```

Comment:
This passes on main, so wasn't actually a problem, but hey I wrote the test before I knew that so may as well keep it.  These kinds of "weak" tests have been helpful as I've done ARC migration.